index.bs

```diff
@@ -351,11 +352,11 @@ entropy introduced by [disallowing direct enumeration of the plugin list](https:
 If so, what kind of sensors and information derived from those sensors does
 this standard expose to origins?
 
-Information from sensors may serve as a fingerprinting vector across origins.
-In addition, sensor also reveals something about my device or environment and
-that fact might be what is sensitive.  In addition, as technology advances,
-mitigations in place at the time a specification is written may have to be
-reconsidered as the threat landscape changes.
+Information from sensors may serve as a fingerprinting vector on the same origin
+or across origins.  In addition, sensor also reveals something about the user’s 
```

Comment:
Maybe s/on the same origin or across origins/on origins/, like in #90?

Reply:
Also, there's some grammar weirdness in the original text too ("sensor also reveals" should probably be "sensors can reveal". Can you fix this while you're in here? Thanks!

Reply:
Fixed "sensors can reveal" comment in latest commit.